modules/nodes-nodes-audit-log-advanced.adoc

```diff
@@ -0,0 +1,137 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-audit-log.adoc
+
+[id="nodes-nodes-audit-log-advanced-{context}"]
+= Advanced Audit
+
+The advanced audit feature provides several improvements over the
+basic audit functionality, including fine-grained events filtering and multiple output back ends.
+
+To enable the advanced audit feature, provide the following values in the
+`openshift_master_audit_config` parameter:
+
+----
+openshift_master_audit_config={"enabled": true, "auditFilePath": "/var/lib/origin/oscp-audit.log", "maximumFileRetentionDays": 14, "maximumFileSizeMegabytes": 500, "maximumRetainedFiles": 5, "policyFile": "/etc/origin/master/adv-audit.yaml", "logFormat":"json"}
+----
+
+[IMPORTANT]
+====
+The policy file *_/etc/origin/master/adv-audit.yaml_* must be available on each master node.
+====
+
+
+The following table contains additional options you can use.
+
+.Advanced Audit Configuration Parameters
+
+[cols="3a,6a",options="header"]
+|===
+| Parameter Name | Description
+
+|`policyFile`
+|Path to the file that defines the audit policy configuration.
+
+|`policyConfiguration`
+|An embedded audit policy configuration.
+
+|`logFormat`
+|Specifies the format of the saved audit logs. Allowed values are `legacy` (the
+format used in basic audit), and `json`.
+
+|`webHookKubeConfig`
+|Path to a `.kubeconfig`-formatted file that defines the audit webhook
+configuration, where the events are sent to.
+
+|`webHookMode`
+|Specifies the strategy for sending audit events. Allowed values are `block`
+(blocks processing another event until the previous has fully processed) and
+`batch` (buffers events and delivers in batches).
+|===
+
+[IMPORTANT]
+====
+To enable the advanced audit feature, you must provide either `policyFile` *or*
+`policyConfiguration` describing the audit policy rules:
+====
+
+.Sample Audit Policy Configuration
```

Comment:
We should start this section of with a block of text that explains the following as examples of how you can alter the auditing rules and behaviors.